# Generative_Protein_Design_Datasets
A repository highlighting and linking existing resources and datasets for generative protein modeling. 


## Protein Sequence Datasets  

- Suzek, Baris E et al. **“UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches.”**  
  Bioinformatics (Oxford, England) vol. 31,6 (2015): 926-32. doi:10.1093/bioinformatics/btu739  
  **Description:** A collection of protein sequence clusters from the UniProt Knowledgebase.  
  [[Paper]](https://academic.oup.com/bioinformatics/article/31/6/926/214968), [[Dataset/Server]](http://www.uniprot.org/uniref)
  
- Bryant, D.H., Bashir, A., Sinai, S. et al. **"Deep diversification of an AAV capsid protein by machine learning."**  
  Nat Biotechnol 39, 691–696 (2021). https://doi.org/10.1038/s41587-020-00793-4  
  **Description:** DL-engineered adeno-associated virus 2 (AAV2) capsid protein variants.  
  [[Paper]](https://www.nature.com/articles/s41587-020-00793-4#Abs1), [[Dataset/Server]](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA673640/)
  
- Jaina Mistry et al. **"Pfam: The protein families database in 2021"**  
  Nucleic Acids Research, Volume 49, Issue D1, 8 January 2021, Pages D412–D419, https://doi.org/10.1093/nar/gkaa913  
  **Description:** A collection of protein families, each represented by multiple sequence alignments and hidden Markov models.  
  [[Paper]](https://academic.oup.com/nar/article/49/D1/D412/5943818?login=true), [[Dataset/Server (Legacy Host)]](http://pfam-legacy.xfam.org/), [[Dataset/Server (Hosted at InterPro)]](https://www.ebi.ac.uk/interpro/)

- Trinquier, J., Uguzzoni, G., Pagnani, A. et al. **"Efficient generative modeling of protein sequences using simple autoregressive models."**
  Nat Commun 12, 5800 (2021). https://doi.org/10.1038/s41467-021-25756-4  
  **Description:** Uses data for five protein families (PF00014,PF00072, PF00076,PF00595,PF13354) from Pfam.  
  [[Paper]](https://www.nature.com/articles/s41467-021-25756-4), [[Dataset/Server]](https://github.com/pagnani/ArDCAData)

- Elodie Laine et al. **"GEMME: A Simple and Fast Global Epistatic Model Predicting Mutational Effects**"  
  Molecular Biology and Evolution, Volume 36, Issue 11, November 2019, Pages 2604–2619, https://doi.org/10.1093/molbev/msz179  
  **Description:** Generates the mutational landscape of a protein given an input alignment    
  [[Paper]](https://academic.oup.com/mbe/article/36/11/2604/5548199), [[Datatset/Server]](http://www.lcqb.upmc.fr/GEMME/download.html)

- Shin, JE., Riesselman, A.J., Kollasch, A.W. et al. **"Protein design and variant prediction using autoregressive generative models."**  
  Nat Commun 12, 2403 (2021). https://doi.org/10.1038/s41467-021-22732-w  
  **Description:** PTEN phosphatase deletions & IGP dehydratase insertions and deletions  
  [[Paper]](https://www.nature.com/articles/s41467-021-22732-w), [[Dataset/Server]](https://github.com/debbiemarkslab/SeqDesign/tree/v3/examples)

- Typhaine Paysan-Lafosse et al. **"InterPro in 2022"**  
  Nucleic Acids Research, Volume 51, Issue D1, 6 January 2023, Pages D418–D427, https://doi.org/10.1093/nar/gkac993  
  **Description**: A database that integrates diverse information about protein families, domains and functional sites, providing a unified view of protein sequence classification and annotation.  
  [[Paper]](https://academic.oup.com/nar/article/51/D1/D418/6814474), [[Datatset/Server]](http://www.ebi.ac.uk/interpro/)

- Milot Mirdita et al. **"Uniclust databases of clustered and deeply annotated protein sequences and alignments"**  
  Nucleic Acids Research, Volume 45, Issue D1, January 2017, Pages D170–D176, https://doi.org/10.1093/nar/gkw1081  
  **Description:** A database of protein sequences clustered at different sequence similarity thresholds, providing a resource to reduce redundancy and improve the efficiency of various bioinformatics analyses  
  [[Paper]](https://academic.oup.com/nar/article/45/D1/D170/2605730?login=true), [[Dataset/Server]](https://uniclust.mmseqs.com/)

- Gilson, Michael K et al. “BindingDB in 2015: A public database for medicinal chemistry, computational chemistry and systems pharmacology.”  
  Nucleic acids research vol. 44, D1 (2016): D1045-53. doi:10.1093/nar/gkv1072  
  **Description:** A database of measured binding affinities, focusing chiefly on the interactions of proteins considered to be drug-targets with small, drug-like molecules.  
  [[Paper]](https://pubmed.ncbi.nlm.nih.gov/26481362/), [[Dataset/Server]](https://www.bindingdb.org/rwd/bind/index.jsp)

- Antje Chang et al. **"BRENDA, the ELIXIR core data resource in 2021: new developments and updates"**  
  Nucleic Acids Research, Volume 49, Issue D1, 8 January 2021, Pages D498–D508, https://doi.org/10.1093/nar/gkaa1025  
  **Description:** A database providing detailed information on enzymes such as function, structure, localization, and application, derived from scientific literature and manual curation.  
  [[Paper]](https://academic.oup.com/nar/article/49/D1/D498/5992283), [[Dataset/Server]](https://www.brenda-enzymes.org/)

- Schmitt, L.T., Paszkowski-Rogacz, M., Jug, F. et al. **"Prediction of designer-recombinases for DNA editing with generative deep learning."**  
  Nat Commun 13, 7966 (2022). https://doi.org/10.1038/s41467-022-35614-6  
  **Description:** 89 recombinase libraries for loxA-1 to loxX-6 target sites  
  [[Paper]](https://www.nature.com/articles/s41467-022-35614-6), [[Dataset/Server]](https://www.ebi.ac.uk/ena/browser/view/PRJEB57361)

- 
  

  

## Protein Structure Datasets
- Ian Sillitoe et al. **"CATH: expanding the horizons of structure-based functional annotations for genome sequences"**  
  Nucleic Acids Research, Volume 47, Issue D1, 08 January 2019, Pages D280–D284, https://doi.org/10.1093/nar/gky1097  
  **Description:** Hierarchical Classification of Protein Structures based on function and evolutionary relationships.  
  [[Paper]](https://academic.oup.com/nar/article/47/D1/D280/5162467), [[Dataset/Server]](http://www.cathdb.info/)

- Cuff, J A, and G J Barton. **“Evaluation and improvement of multiple sequence methods for protein secondary structure prediction.”**  
  Proteins vol. 34,4 (1999): 508-19. doi:10.1002/(sici)1097-0134(19990301)34:4<508::aid-prot10>3.0.co;2-4  
  **Description:** Dataset for Secondary Structure Prediction  
  [[Paper]](https://pubmed.ncbi.nlm.nih.gov/10081963/), [[CB513 Dataset]](https://github.com/taneishi/CB513_dataset)

- Kryshtafovych, A, Schwede, T, Topf, M, Fidelis, K, Moult, J. **"Critical assessment of methods of protein structure prediction (CASP)—Round XIII."**    
  Proteins. 2019; 87: 1011– 1020. https://doi.org/10.1002/prot.25823  
  **Description:** A collection of predicted structures for sequences  
  [[Paper]](https://onlinelibrary.wiley.com/doi/10.1002/prot.25823), [[Datatset/Server]](https://predictioncenter.org/casp13/)

- [Protein Data Bank](https://www.rcsb.org/)

- John-Marc Chandonia et al. **"SCOPe: classification of large macromolecular structures in the structural classification of proteins—extended database."**  
  Nucleic Acids Research, Volume 47, Issue D1, 08 January 2019, Pages D475–D481, https://doi.org/10.1093/nar/gky1134  
  **Description:** Database of known protein structures classified based on their structural and evolutionary relationships  
  [[Paper]](https://academic.oup.com/nar/article/47/D1/D475/5219094), [[Dataset/Server]](http://scop.berkeley.edu/)  

- 
  

## Protein-Protein Interaction Datasets
-

## Protein-Ligand Datasets
